server/app/lib/actions/fusor/configure_host_groups.rb

```diff
@@ -61,50 +61,42 @@ def find_or_ensure_hostgroup(deployment_name, organization_id, lifecycle_environ
 
           lifecycle_environment = ::Katello::KTEnvironment.find(lifecycle_environment_id)
           content_view_puppet_environment = content_view.puppet_env(lifecycle_environment)
+          puppet_environment = content_view_puppet_environment.puppet_environment
 
           if puppet_class_settings = hostgroup_settings[:puppet_classes]
             puppet_classes = Puppetclass.where(:name => puppet_class_settings.map{ |c| c[:name] }).
                 joins(:environment_classes).
-                where("environment_classes.environment_id in (?)",
-                      content_view_puppet_environment.puppet_environment.id).uniq
+                where("environment_classes.environment_id in (?)", puppet_environment.id).uniq
```

Comment:
@pitr-ch, thanks! I will update and test with a future PR.  I think we have a couple of instances where this format is used.